seed_db.sh

```diff
@@ -0,0 +1,217 @@
+
+
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (1, 'GERMAN', 'REVIEW', 3);"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (1, 'ENGLISH_UK', 'REVIEW', 3);"
+
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (2, 'ARABIC', 'REVIEW', 2);"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (2, 'SWAHILI', 'REVIEW', 3);"
+
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (3, 'MANDARIN', 'REVIEW', 3);"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (3, 'ENGLISH_UK', 'START', 4);"
+
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (4, 'GERMAN', 'REVIEW', 3);"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (4, 'ENGLISH_UK', 'START', 7);"
+
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (5, 'GERMAN', 'START', 1);"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (5, 'ENGLISH_UK', 'START', 2);"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (5, 'PORTUGUESE', 'START', 3);"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (5, 'DUTCH', 'START', 4);"
+
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (6, 'ENGLISH_US', 'START', 3);"
+
+
+    # Story content 
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "ALTER SEQUENCE story_contents_id_seq RESTART WITH 1;"
+
+    for STORY_ID in 1 2 3 4 5 6 7
+    do 
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_contents (story_id, line_index, content) VALUES ('$STORY_ID', 1, '“Every two weeks I went to a meeting with them, and in my room here I covered pages with writing. 
```

Comment:
Hey, I'm using some of the script manually atm: 

` ('$STORY_ID', 1, '“Every ` -> what's the `“` for?